docker/dockerfiles/allin1/allin1-ubuntu.Dockerfile

```diff
@@ -60,6 +61,7 @@ RUN mkdir -p $DEPLOYDIR/starrocks/fe/meta && mkdir -p $DEPLOYDIR/starrocks/be/st
 COPY *.sh $DEPLOYDIR
 RUN chmod +x *.sh
 
+# Copy config files
```

Comment:
a little strange on `line 66`, why use `cat` instead of `mv`?

Reply:
to avoid the default generated config values, I append the customized configs to the end of  of .conf file.
Yep, agree it might not be the idea but thought it good enough for the purpose of allin1 test docker image.

E.g. here are the defaulted generated values `be.conf`

```
# INFO, WARNING, ERROR, FATAL
sys_log_level = INFO

# ports for admin, web, heartbeat service 
be_port = 9060
be_http_port = 8040
heartbeat_service_port = 9050
brpc_port = 8060
```

Reply:
may not be a good idea. User provided custom configuration should be a full functional conf without a `base` version. Otherwise it is not clear that who takes the precedence.

Reply:
Did some experiment today. 
There are other system wide configs(e.g. `JAVA_OPTS`,  ) in Config.java that needs to be put inside the .conf file to make the service start properly. Simply replace the pre-generated .conf files with my custom .conf files won't work.
It's also not realistic to maintain a complete copy of the config values in the custom .conf file as it could change over time. So I think the most lightweight way is to just append what I want to override at the end of the .conf file.

Also confirmed by @chaoyli , if we append the custom config values at the end of the .conf file, it will only override the config values that are appended., the rest of the configs will stay as the default values.

Reply:
ok